nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2944,95 +3048,95 @@ at this time.
 
```

Comment:
In line `3039`, the *Note Well* should be changed to *NOTE:* for consistency.

Reply:
I saw that and didn't want to change it, not knowing the author's intent.  But, if you agree that this is worthy to change for consistency, then I am in agreement also.